tests/vxlan/__init__.py

```diff
@@ -9,61 +9,6 @@
 
 logger = logging.getLogger(__name__)
 
-def pytest_addoption(parser):
```

Comment:
Why do we have vxlan change in this PR? Suggest to separate if not part of this feature.  @theasianpianist for visibility

Reply:
This change is to resolve the `--num-routes` option conflicting issue. Please refer to https://github.com/Azure/sonic-mgmt/pull/2344#discussion_r508864353

Reply:
The `--num_routes` option is used for route tests and vxlan tests. Currently, vxlan test put `pytest_addoption` in __init__.py. This will be seen by pytest during initialization and cause conflict if any other tests try to add `--num_routes` option. I moved `pytest_addoption` to conftest.py to resolve conflict.

Reply:
@prsunny I originally put these in `__init__.py` because we had a bug that caused CLI options defined in test specific `conftest.py` files to not be recognized. I believe @wangxin has since fixed the issue, so this should be fine.